website/src/components/broadcast/PlayerHero.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="hero-card d-flex flex-column flex-center" v-if="hero">
+    <div class="hero-name d-flex flex-row text-center font-weight-bold"><span v-if="roleIcon" class="icon" v-html="roleIcon"></span> {{heroName}}</div>
+    <div class="hero-role d-flex flex-row text-center">{{heroRole}}</div>
+    <div class="hero-portrait d-flex flex-row flex-center"><img :src="heroPortraitURL"/></div>
+  </div>
+</template>
+
+<script>
+
+export default {
+    name: "PlayerHero",
+    props: ["hero"],
+    computed: {
+        heroName() {
+            if (!this.hero?.name) return null;
+            return this.hero.name;
+        },
+        heroRole() {
+            if (!this.hero?.name) return null;
+            return this.hero.role;
+        },
+        roleIcon() {
+            switch (this.heroRole) {
+            case "Tank":
+                return "<i class=\"fas fa-shield-alt\"></i>";
+            case "Support":
+                return "<i class=\"fas fa-plus-circle\"></i>";
+            case "DPS":
+                return "<i class=\"fas fa-crosshairs\"></i>";
+            }
+            return null;
+        },
+        heroPortraitURL() {
+            if (!this.hero?.main_image) return null;
+            return this.hero?.main_image[0].url;
+        }
```

Comment:
Did you know you can use optional chaining with square brackets? I didn't until recently. It looks really dumb: `?.[0]`. There's a helper function somewhere that deals with Airtable images but to be honest I don't care too much for it.

```suggestion
        heroPortraitURL() {
            return this.hero?.main_image?.[0]?.url;
        }
```

Reply:
logic changes in here 8a1a202